spec/features/idv/steps/in_person/address_spec.rb

```diff
@@ -134,7 +134,7 @@
     end
   end
 
-  context 'Validation' do
+  context 'validation' do
```

Comment:
It looks we only check validation for zip code. Do you think we should check that required displays when the inputs are blank?